src/repart/repart.c

```diff
@@ -6098,7 +6098,8 @@ static int context_mkfs(Context *context) {
                         return r;
 
                 r = make_filesystem(partition_target_path(t), p->format, strempty(p->new_label), root,
-                                    p->fs_uuid, arg_discard, /* quiet = */ false,
+                                    p->fs_uuid, arg_discard,
+                                    /* quiet = */ streq(p->format, "erofs") && !DEBUG_LOGGING,
```

Comment:
uh, please jus move this into make_filesystem(). don't repeat this at all three places...

it's not tha twe call make_filesystem for erofs anywhere else but from repart, hence it really should be in generic code, in aprticular as this is stuff to remove sooner or later.